data/mods/ssb/random-teams.ts

```diff
@@ -427,6 +427,41 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'Class Change',
 		evs: {hp: 4, spa: 252, spe: 252}, ivs: {atk: 0}, nature: ['Modest', 'Timid'],
 	},
+	'yuki-Cleric': {
+		species: 'Pikachu-PhD', ability: 'Triage', item: 'Light Ball', gender: 'F',
+		moves: ['Parabolic Charge', 'Wish', 'Baton Pass'],
+		signatureMove: 'Class Change',
+		evs: {hp: 4, spa: 252, spe: 252}, ivs: {atk: 0}, nature: ['Modest', 'Timid'],
```

Comment:
Honestly I think in this scenario the nature  should remain constant

Reply:
Well, the `nature` attribute is a mandatory part of an `SSBSet`, so something has to be specified. However, I can make `changeSet` not modify the nature when the set is yuki.

Reply:
i can figure something out